components/help-and-support/SearchTopicsSection.js

```diff
@@ -154,6 +154,7 @@ const SearchTopics = () => {
               setIsLoading(true);
               debouncedSearch(query);
             }}
+            onClearFilter={() => setSearchQuery('')}
```

Comment:
Strange bug on this one: I get redirected to the search page when I clear the input

To reproduce:
1. Go to search page
2. Make a search, clear it
3. Click on "Docs & Help" in the footer
4. Make a search, clear it
5. See bug

https://github.com/opencollective/opencollective-frontend/assets/1556356/3426e33c-f336-4b1a-ba47-9298b941a3df